config/locales/es.yml

```diff
@@ -1564,6 +1564,7 @@ step_indicator.status.complete: Completado
 step_indicator.status.current: Este paso
 step_indicator.status.not_complete: No completado
 time.am: a. m.
+time.formats.standard_long: '%-d de %B de %Y'
```

Comment:
should we use this for event date if the current one is wrong?

Reply:
Okay, I wasn't sure if the change should app wide or not which is why I created a new translation.

Reply:
We have `time.formats.full_date`, `time.formats.event_date`, which both have some overlap with each other and `time.formats.standard_long`.

There are also other dates for IPP that use `event_date`, ex: https://github.com/18F/identity-idp/blob/a534a340f75beeed20d043cb6c6860a6bdad091a/app/presenters/account_show_presenter.rb#L77-L79, so I think it's probably worth just updating `event_date` for this.